cor_tabular/code/make_table_all.R

```diff
@@ -396,7 +405,7 @@ tab_dm <- full_join(
               names_sort = T,
               values_from = c(rslt)) %>%
   mutate(Characteristics = factor(subgroup_cat,
-                                  levels=c("Age $\\geq$ 65","Age $<$ 65",
+                                  levels=c("Age $<$ 65", "Age $\\geq$ 65",
```

Comment:
Is the change of ordering here correct?

Reply:
Yes, these are for the subgroup "Age" not "Age, Risk for Severe COVID". Should I remove "Age" as it became somehow redundant now?

Reply:
Whatever you think is better. Just wanted to make sure the ordering was correct!

Reply:
Let me know if you want to make another change; otherwise I will merge. 

Thanks!!

Reply:
I think no further changes needed :) 
Thank you!